sftp/1.3/modules/ROOT/edits/sftp-write.adoc

```diff
@@ -0,0 +1,75 @@
+= Write a File Using the SFTP Connector - Mule 4
+:page-aliases: connectors::sftp/sftp-write.adoc
+
+Anypoint Connector for SFTP (SFTP Connector) Write operation writes content into the given path on demand.
+
+== Configure the Write Operation in Studio
+
+To add and configure the Write operation in Studio, follow these steps:
+
+. In the *Mule Palette* view, search for `sftp` and select the *Write* operation.
+. Drag the *Write* operation onto the Studio canvas.
+. In the *General* tab of the operation configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. Specify the connection information and click *OK*.
+. In the *General* tab, set the *Path* field to `output.csv` to set the path of the file to write. +
+By default, the connector writes whatever is in the message payload.
+
+The following screenshot shows the *Write* operation configuration:
+
+.Write operation configuration
+image::sftp-write-operation-1.png[Write operation configuration in Studio]
+
+In the XML editor, the `<sftp:write>` configuration looks like this:
+[source,xml,linenums]
+----
+<sftp:write path="output.csv" />
+----
+
+
+== Use Embedded DataWeave Transformations
+
+If you place a Transform component before the Write operation, the message payload changes and that impacts the operation that is placed after the Write operation.
+So if the payload is in a different format and you need to transform it before writing it, you can place the transformation inside the *Content* parameter of the Write operation to generate content to be written without producing a side effect on the message in transit.
+
+The following example places the DataWeave transformation in the *Content* parameter:
+
+.Write operation configuration with Content parameter
+image::sftp-write-operation-2.png[Write operation configuration in Studio with Content parameter]
+
+In the XML editor, the configuration looks like this:
+[source,xml,linenums]
+----
+<sftp:write path="output.csv">
+  <sftp:content>#[%dw 2.0
+                   output application/csv
+                   ---
+                   payload.customers.email
+                  ]
+  </sftp:content>
+</sftp:write>
+----
+
+== Write into Directories
+
+The Write operation fails by default if you set non-existent directories in the *Path* parameter. Therefore, you can set the *Crete parent directories* parameter to `True (Default)` to automatically create any missing directories.
```

Comment:
Per Webster's dictionary...:
```suggestion
Because the Write operation fails by default if you set nonexistent directories in the *Path* parameter, set the *Create parent directories* parameter to `True (Default)` to automatically create any missing directories.
```